plugins/module_utils/podman/podman_pod_lib.py

```diff
@@ -573,7 +573,13 @@ def running(self):
         """Return True if pod is running now."""
         if 'status' in self.info['State']:
             return self.info['State']['status'] == 'Running'
-        return self.info['State'] == 'Running'
+        if isinstance(self.info['State'], str):
+            return self.info['State'] == 'Running'
+        # older podman versions (1.6.x) don't have status in 'podman pod inspect'
+        # if other methods fail, use 'podman pod ps'
+        ps_info = self.get_ps()
+        if 'status' in ps_info:
+            return ps_info['status'] == 'Running'
```

Comment:
In this case it will return `None` if something went wrong, which is same as `False`. Let's keep the previous default, and keep return the same: `return self.info['State'] == 'Running'` and then no need to check it in lines 576-577 above.

Reply:
I changed back to the previous default.